.github/workflows/lint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
               echo "cbf=false" >> $GITHUB_ENV
             fi
         - name: Commit changes to PR
-          if: env.cbf == 'true'
```

Comment:
You should probably merge https://github.com/pantheon-systems/pantheon-mu-plugin/pull/86 first